servers/cromwell/jobs/controllers/aggregation_controller.py

```diff
@@ -0,0 +1,21 @@
+import connexion
+from jobs.models.aggregation_response import AggregationResponse
+from datetime import date, datetime
+from typing import List, Dict
+from six import iteritems
+from ..util import deserialize_date, deserialize_datetime
+
+
+def get_job_aggregations(timeFrame, projectId=None):
+    """
+    Query for aggregated jobs in the given time frame.
```

Comment:
nit: We use a different comment syntax - please update: https://github.com/DataBiosphere/job-manager/blob/master/servers/dsub/jobs/controllers/jobs_controller.py#L76

Reply:
Style fixed.